__tests__/pet.test.js

```diff
@@ -0,0 +1,112 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  const pet = new Pet("Fido");
+  it("returns an object", () => {
+    expect(new Pet()).toBeInstanceOf(Object);
+  });
+  it("renames pet", () => {
+    expect(pet.name).toBe("Fido");
+  });
+  it("has an initial age of 0", () => {
+    expect(pet.age).toBe(0);
+  });
+  it("has an initial hunger of 0", () => {
+    expect(pet.hunger).toBe(0);
+  });
+  it("has an intial fitness of 10", () => {
+    expect(pet.fitness).toBe(10);
+  });
+});
+
+describe("growUp", () => {
+  const pet = new Pet("Fido");
+  it("thows an error if pet is not alive", () => {
+    pet.age = 30;
+    expect(() => pet.feed()).toThrow("Your pet is no longer alive :(");
+  });
+  it("increments age by 1, hunger by 5, decrements fitness by 3", () => {
+    pet.age = 0;
+    pet.growUp();
+    expect(pet.age).toEqual(1);
+    expect(pet.hunger).toEqual(5);
+    expect(pet.fitness).toEqual(7);
+  });
+});
+
+describe("walk", () => {
+  const pet = new Pet("Fido");
+  it("increases fitness level", () => {
+    pet.fitness = 4;
+    pet.walk();
+    expect(pet.walk).toBeInstanceOf(Function);
+    expect(pet.fitness).toBeLessThanOrEqual(10);
+    expect(pet.fitness).toBeGreaterThanOrEqual(0);
```

Comment:
If youre checking for a specific value as on line 45, why check for 2 other ranges like this?